tests/trx_generator/trx_generator.hpp

```diff
@@ -15,8 +18,66 @@ namespace eosio::testing {
       fc::crypto::private_key _signer;
    };
 
+   struct action_pair_w_keys {
+      action_pair_w_keys(eosio::chain::action first_action, eosio::chain::action second_action, fc::crypto::private_key first_act_signer, fc::crypto::private_key second_act_signer)
+            : _first_act(first_action), _second_act(second_action), _first_act_priv_key(first_act_signer), _second_act_priv_key(second_act_signer) {}
+
+      eosio::chain::action _first_act;
+      eosio::chain::action _second_act;
+      fc::crypto::private_key _first_act_priv_key;
+      fc::crypto::private_key _second_act_priv_key;
+   };
+
+   struct account_name_generator {
+      account_name_generator() : _name_index_vec(ACCT_NAME_LEN, 0) {}
+
+      const char* CHARMAP = "12345abcdefghijklmnopqrstuvwxyz";
+      const int ACCT_NAME_CHAR_CNT = 31;
+      const int ACCT_NAME_LEN = 12;
+      const int MAX_PREFEX = 960;
```

Comment:
Reserving the first 2 characters in the 12 char account name to identify the transaction generator sending the trx to create the new account.  So 31 ^ 2 gives 961 (so w/0 based index, 960) since we may want more than 31 generators but shouldn't ever want or need > 961.  This provides an easy way to deduplicate the names being generated by the parallel trx generators.

Reply:
Please add this info to a comment on the 960.

Reply:
Addressed: dd4fb9143fe